tests/framework/shader_templates.h

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright (c) 2015-2023 The Khronos Group Inc.
+ * Copyright (c) 2015-2023 Valve Corporation
+ * Copyright (c) 2015-2023 LunarG, Inc.
+ * Copyright (c) 2015-2023 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+#pragma once
+
+static const char kMinimalShaderGlsl[] = R"glsl(
+    #version 460
+    void main() {}
+)glsl";
+
+static const char kVertexMinimalGlsl[] = R"glsl(
+    #version 460
+    void main() {
+       gl_Position = vec4(1);
+    }
+)glsl";
+
+static const char kVertexPointSizeGlsl[] = R"glsl(
+    #version 460
+    out gl_PerVertex {
+        vec4 gl_Position;
+        float gl_PointSize;
+    };
+    void main() {
+        gl_Position = vec4(1);
+        gl_PointSize = 1.0;
+    }
+)glsl";
+
+static char const kGeometryMinimalGlsl[] = R"glsl(
+    #version 460
+    layout(triangles) in;
+    layout(triangle_strip, max_vertices=3) out;
+    void main() {
+       gl_Position = vec4(1);
+       EmitVertex();
+    }
+)glsl";
+
+static char const kGeometryPointSizeGlsl[] = R"glsl(
+    #version 460
+    layout (points) in;
+    layout (points) out;
+    layout (max_vertices = 1) out;
+    in gl_PerVertex {
+        vec4 gl_Position;
+        float gl_PointSize;
+    };
+    void main() {
+       gl_Position = vec4(1);
+       gl_PointSize = 1.0;
+       EmitVertex();
+    }
+)glsl";
+
+static const char kTscMinimalGlsl[] = R"glsl(
```

Comment:
```suggestion
static const char kTesselationControlMinimalGlsl[] = R"glsl(
```
Given the other shader template names, if I was to intuitively try find a tessellation shader template, I would look for this name.
But yeah it is longer...

Reply:
I would be lying if 90% of this decision was because I didn't want to look up how to spell `Tessellation`... will fix though